src/status_im/contexts/chat/messenger/composer/effects.cljs

```diff
@@ -114,9 +114,9 @@
   [{:keys [input-ref]}
    {:keys [text-value saved-cursor-position cursor-position]}
    {:keys [edit input-with-mentions]}
-   messages-list-on-layout-finished?]
+   calculations-complete?]
```

Comment:
what calculations? is it possible to give some context? idk in the name or in the comment

Reply:
Hi @flexsurfer, Thank you very much for reviewing PR.

Here, by calculations we mean that chat screen layout calculations complete.

what do you recommend, should we rename it to `chat-screen-layout-calculations-complete?` or just add a comment where this shared value is created?

(this shared value is used across other files, so here big name might affect formatting and readability)

Reply:
Probably `chat-screen-layout-calculations-complete?` will ok, will change it.